### DISCUSSION TITLE
timestamp error - or ARMv7 docker version issue [Feb 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5079] [Discord Threads]
> 🙋‍♂️ **Pierric**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Pierric**  
#### I'm wondering if I have the right image to be honest... it does seem to be the correct latest version from 1 month ago, however the package.json inside the container mentions version 0.84.12 (running on ARMv7)  
#  
> 🙋‍♂️ **Pierric**  
#### yup, just popped nocodb onto a x86 platform and it looks so different now... my Rapsberry pi is stuck on an old version, despite pulling the latest image in different ways 🤔  
#  
> 🙋‍♂️ **Pierric**  
#### never mind, this is because I have /usr/src/app mounted as a volume. I guess installation guidelines have changed since my first install, and that was a breaking change.  
#  
> 🙋‍♂️ **wkw**  
#### If you are using latest tag, you need to remove your local image first and pull the updated one. Otherwise it will reuse the old image. For the version, you can verify it by hovering the icon on the top left side.   
####    
#### For the time error, based on the error message you provided, it looks like it is expected as you pass the date value to a column with Time type (expected value should be something like 03:30).    
####    
#### I just had a quick test on datetime & time columns and they are working fine. (latest version - 0.100.2)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1060592434914004993)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
